### PR TITLE
hwdb: add CST Laser Trackball

### PR DESCRIPTION
**Pull request by prumian**
Add DPI and frequency for CST Laser Trackball. Tested and proved working on a local system.

### CODE REVIEW

File: hwdb.d/70-mouse.hwdb [hunk 1]
```diff
@@ -166,6 +166,14 @@ mouse:bluetooth:v05acp030d:name:*:*
 mouse:usb:v04f2p0963:name:Chicony 2.4G Multimedia Wireless Kit:*
  MOUSE_DPI=1000@142
 
+##########################################
+# CST
+##########################################
+
+# CST Laser Trackball
+mouse:usb:v5332p1400:name:Clearly Superior Technologies. CST Laser Trackball:*
+ MOUSE_DPI=400@1000 800@1000 *1600@1000
```
Comment (poettering) [inline hwdb.d/70-mouse.hwdb]: if this is a trackball, it should probably also set `ID_INPUT_TRACKBALL=1`?

Comment (prumian) [reply hwdb.d/70-mouse.hwdb]: @poettering It's being caught by general rule for trackballs in lines 143-149, so probably not needed explicitly?

Comment (whot) [reply hwdb.d/70-mouse.hwdb]: yep, not needed explicitly

@prumian: does this thing really ship with 1600 out of the box? the bits I found online say it's button-selectable, is this a hw button or a software-configured button?

Comment (prumian) [reply hwdb.d/70-mouse.hwdb]: @whot: it's selectable by holding right+middle button, it goes 400-800-1600 with every click. Super hard to tell which resolution it came from factory though. I'll try to find the manual...

Comment (prumian) [reply hwdb.d/70-mouse.hwdb]: Strangely, the manual says 800DPI is the default. I have updated the PR.

**Review by whot — APPROVED** [review]

File: hwdb.d/70-mouse.hwdb (hunk 1 above)
Comment (whot) [reply hwdb.d/70-mouse.hwdb]: fwiw, 800 by default makes sense, that and 1000dpi are the two most common ones. Anything else is too fast by default, leading to angry customers screaming at the hotlines (or more likely these days: reddit :). Thanks for the update, this looks good now.

**Pull request merged**